### ISSUE TITLE
Training log of baseling / ASFF model？

### ISSUE THREAD

**Issue by zeyu-liu** [ruinmessi/ASFF#44]
Could you release the training log of baseline model & ASFF model? I tried to train a baseline model with less epochs (30) for quick testing, but I found that the validation mAP is quite low (0.08 at epoch 16). And the conf loss / clf loss seems not converge (near 100...). Is this normal?

**Comment by ruinmessi** [comment]
Sure, I will find the log this weekend. I think I have the log of the baseline model and ASFF model, but I am not sure the log of ASFF* still exists. And I think your training loss and val mAP is not so bad (the val mAP at epoch30 is around 0.10-0.15)

**Comment by ruinmessi** [comment]
@zeyu-liu  The training logs are now available

**Comment by zeyu-liu** [comment]
> @zeyu-liu The training logs are now available

Thx!

**Comment by kareemalsawah** [comment]
Is it possible to upload the training logs to another platform other than baidu? I am not able to download the logs from it.
